src/actions/mintEditionFromMaster.ts

```diff
@@ -46,6 +46,7 @@ export const mintEditionFromMaster = async (
     TOKEN_PROGRAM_ID,
     masterEditionMint,
     wallet.publicKey,
+    true
```

Comment:
Under what circumstances would the `wallet.publicKey` not be on curve?

Reply:
Certain wallet addresses are not on Curve due to how they're architected. For Example, [Blocto Wallet](https://solana-labs.github.io/wallet-adapter/classes/_solana_wallet_adapter_blocto.BloctoWalletAdapter.html#sendTransaction).

Regardless, this is a call to `getAssociatedTokenAddress`, not a creation. So I think it's unnecessary to let the function call default to `false`.